src/components/map/Map.js

```diff
@@ -120,70 +124,93 @@ const Map = ({ deals }) => {
+            className="bg-white border-button-blue font-bold text-button-blue border-2 mt-2 p-2 w-fit rounded-lg shadow-md z-50"
+            onClick={() => {
+              map.panTo({ lat: center.lat, lng: center.lng });
+              map.setZoom(13);
+            }}
+          >
+            Reset View
+          </button>
+      </div>
+      {selectedMark  && selectedPlace !== -1 ? (
+      <>
+        <div className="absolute left-0 bottom-2 md:bottom-32 flex px-4 justify-center w-full">
+          <div className="bg-white mt-2 p-3 w-full lg:w-3/4 rounded-lg shadow-md z-50">
               <div>
-                All deals available:{" "}
-                {selectedMark.deals.map((deal) => {
-                  return (
-                    <Link to={`/place/${deal.placeId}?deal_id=${deal._id}`}>
-                      <h3 className="hover:underline">- {deal.title}</h3>
-                    </Link>
-                  );
-                })}
-              </div>
-              <hr className="my-1" />
-              <div className="flex justify-end">
-                <Link
-                  to={`/place/${selectedMark.placeId}`}
-                  className="italic text-hoboken-blue text-sm hover:underline"
-                >
-                  See all deals...
-                </Link>
+                <div>
+                  <div className="flex justify-between">
+                    <button
+                      onClick={() => {
+                        map.panTo({ lat: selectedMark.lat - 0.0008, lng: selectedMark.lng });
+                        if (map.zoom !== 17) {
+                          map.setZoom(17);
+                        }
+                      }}
+                      className="text-button-blue hover:underline flex items-center"
+                    >
+                      <CenterLocationIcon tw="h-4" />
+                      Jump To Location
+                    </button>
+                    <button
+                      className="flex items-center text-gray-500 z-40 lg:text-sm"
+                      onClick={() => {
+                        closeButton()
+                      }}
+                    >
+                      <XIcon tw="h-6 w-6" />
+                    </button>
+                  </div>
+                <hr className="my-1" />
+                </div>
+                <div className="flex items-center justify-space-between pt-1">
+                  <Link
+                    to={`/place/${selectedMark.placeId}`}
+                    className="flex items-center gap-x-2 flex-wrap"
+                  >
+                    <h1 className="text-2xl font-bold">
+                      {selectedMark?.placeName}
+                    </h1>
+                    <RightArrowIcon />
+                  </Link>
+                  <div className="flex ml-auto">
+                    <a
+                      href={`https://www.google.com/maps/dir//${selectedMark.placeName}, ${selectedMark.address}`}
```

Comment:
since we use this formatting in directions for this we should prob change the place screen to do the same